app/Http/Controllers/MissingController.php

```diff
@@ -120,16 +120,16 @@ public function index_gamefiles(){
     }
 
     //Spiele mit fehlender Spielebeschreibung anzeigen
-    public function index_gamedesc(){
+    public function index_gamedesc() {
```

Comment:
__[code_sniffer]__
* Method name "MissingController::index_gamedesc" is not in camel caps format
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->